src/commands/deploy/deploy.ts

```diff
@@ -338,7 +338,18 @@ async function deployToSasjs(target: Target, isLocal: boolean, sasjs?: SASjs) {
 
   let authConfig
   if (!sasjs) {
-    ;({ sasjs, authConfig } = await getSASjsAndAuthConfig(target, isLocal))
+    sasjs = new SASjs({
+      serverUrl: target.serverUrl,
+      appLoc: target.appLoc,
+      serverType: target.serverType,
+      httpsAgentOptions: target.httpsAgentOptions,
+      debug: true,
+      useComputeApi: true
```

Comment:
is this part relevant for SASjs server type?

Reply:
removed